datalad/distribution/install.py

```diff
@@ -178,106 +191,123 @@ def __call__(
+                        # the very top, otherwise we are not able to order a global
+                        # "ignore-and-keep-going"
+                        on_failure='ignore',
+                        return_type='generator',
+                        result_xfm=None,
+                        **common_kwargs):
+                    # no post-processing of the installed content on disk
+                    # should be necessary here, all done by code further
+                    # down that deals with an install from an actuall `source`
+                    # any necessary fixes should go there too!
+                    # TODO generator: possibly adjust refds
+                    yield r
+
+            # 2. one or more dataset content paths
             if to_get:
                 lgr.debug("Install passes into get %d items", len(to_get))
                 # all commented out hint on inability to pass those options
                 # into underlying install-related calls.
                 # Also need to pass from get:
                 #  annex_get_opts
 
-                # TODO generator
-                # this is not just about datasets
-                # for not limit to not overwhelm poor install
-                get_results = Get.__call__(
-                    to_get,
-                    # description=description,
-                    # if_dirty=if_dirty,
-                    # save=save,
-                    # git_clone_opts=git_clone_opts,
-                    # annex_init_opts=annex_init_opts,
-                    # TODO stupid in general, but install is not a generator yet
-                    on_failure='ignore',
-                    **common_kwargs
-                )
-                # TODO generator
-                # pass through `get` errors by re-yielding
-                #exc_str_ = ': ' + exc_str(exc) if exc.results else ''
-                installed_datasets = [r['path'] for r in get_results
-                                      if r.get('type') == 'dataset' and r['status'] in ('ok', 'notneeded')]
-                failed = [r['path'] for r in get_results
-                          if r['status'] in ('impossible', 'error')]
-                if failed:
-                    lgr.warning("Some items failed to install: %s", failed)
-                failed_items.extend(failed)
-
-                # compose content_by_ds into result
-                for dspath in installed_datasets:
-                    ds_ = Dataset(dspath)
-                    if ds_.is_installed():
-                        installed_items.append(ds_)
-                    else:
-                        lgr.warning("%s was not installed", ds_)
-
-            return Install._handle_and_return_installed_items(
-                ds, installed_items, failed_items, save)
+                for r in Get.__call__(
+                        to_get,
+                        # TODO should pass-through description, not sure why disabled
```

Comment:
I think it wasn't actually disabled, but outcommented as a reminder to implement that option. And the destiny of a reminder is to become a forgotten artifact. ;-)